frontend/packages/ceph-storage-plugin/src/components/bucket-class/wizard-pages/namespace-store-pages/multi-namespace-store.tsx

```diff
@@ -93,7 +93,7 @@ export const MultiNamespaceStorePage: React.FC<MultiNamespaceStoreProps> = React
           </Title>
           <p className="nb-create-bc-step-page-form__element--light-text">
             {t(
-              'ceph-storage-plugin~Select a single namespace store, defines the write target of the namespace bucket',
+              'ceph-storage-plugin~Select a single NamesapceStore which defines the write targets of the namespaced bucket',
```

Comment:
Select a single Namespace store that defines the write targets of the namespace bucket.